.github/workflows/playwright.yml

```diff
@@ -0,0 +1,46 @@
+name: Playwright Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  e2e-tests:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        env:
+          PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: "true"
+        with:
+          node-version: "14.x"
+          args: install
+
+      - name: Install dependencies
+        uses: borales/actions-yarn@v2.3.0
```

Comment:
@bumi also getting this:
```sh
orales/actions-yarn@v2.3.0 is not allowed to be used in getAlby/lightning-browser-extension. Actions in this workflow must be: within a repository owned by getAlby, created by GitHub, or matching the following: peter-evans/create-or-update-comment@v1, peter-evans/find-comment@v1, release-drafter/release-drafter@v5, zdurham/s3-upload-github-action@master, mujo-code/puppeteer-headful@v2.
```

Reply:
added `borales/actions-yarn` to the list (and allow verified marketplace actions now) 
I've limited it for additional security reasons.

Reply:
Hm, it's still not running:  
https://github.com/getAlby/lightning-browser-extension/actions/runs/2256837754  
Any idea?

Reply:
Let me check if I can add missing ones

Reply:
It's runing.  
Needed to be `borales/actions-yarn@v2.3.0` instead of `borales/actions-yarn` 🤦‍♂️